src/main/java/seedu/address/logic/commands/ShowCommand.java

```diff
@@ -129,49 +130,61 @@ private String getUniqueCategoryInputs(Model model) {
         }
     }
 
-    private Set<String> getUniqueNameInputs(ObservableList<Person> ol) {
+    private List<String> getUniqueNameInputs(ObservableList<Person> ol) {
         return ol.stream()
-                .map(x -> x.getName().toString()).collect(Collectors.toSet());
+                .map(x -> x.getName().toString()).distinct().sorted().collect(Collectors.toList());
     }
```

Comment:
Agreed! This will make the output more determinate, which improves testability.

However, take note that this may lead to O(nlogn) runtime since distinct() has a runtime complexity of O(n) whereas sorted() has a runtime complexity of O(nlogn).
This would cause the getUniqueNameInputs() method to take longer than before, when it has O(n) runtime instead.